src/components/reusable/Table.js

```diff
@@ -1,6 +1,10 @@
 import React from "react";
+import { BsCaretDownFill, BsFillCaretUpFill } from "react-icons/bs";
+
+const Table = ({ data, header, viewBacktest }) => {
```

Comment:
This shouldnt be called viewBacktest, it should just be an `onItemPress` not specifi to backtest